packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue

```diff
@@ -134,7 +135,8 @@ export default defineComponent({
       }
 
       try {
-        await this.graphClient.users.getUser(this.user.onPremisesSamAccountName)
+        const client = this.clientService.graphAuthenticated
+        await client.users.getUser(this.user.onPremisesSamAccountName)
```

Comment:
is the response irrelevant here? I know, was like that before, but still wondering now...

Reply:
Got me wondering as well, but now I remember: We request the user here to validate that the username does not yet exist. If the request succeeds, we throw a validation error. So the response itself is irrelevant.

Reply:
Could you add a comment in the code explaining this?